lib/faraday/tracer.rb

```diff
@@ -43,13 +46,12 @@ def call(env)
         span.set_tag('http.status_code', response.status)
 
         if response.status >= 500
-          span.set_tag('error', true)
-          span.log_kv(event: 'error', message: response.body.to_s)
+          error = HTTPInstrumentationError.new response.body.to_s
```

Comment:
This one seems funny to me - I don't think we did it this way in other languages.  It's ok not to have all the `sfx.*` fields after all, and this one only seems to have a message.  Perhaps `kind=HTTP status <status>`?

Reply:
I wanted to strictly translate old logs to new attributes as part of this work to keep the scope down but you're right, this doesn't make much sense. I'll update it to explicitly set `error=True` and may be add `sfx.error.message`.

Reply:
but now you added `HTTPInstrumentationError` and you're not using it